canonical-web-and-design/ubuntu.com
# CUBE: Code review changes

## Done

- Change `p-cube-image` class to `p-floating-image` to make class more reusable 
- Fix spacing issues in `.scss` files
- Make some copy changes
- use `{{ modules|length }}` instead of 15 incase this changes in the future 
- Add `u-no-margin--bottom` class to button instead of using css

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/cube and http://0.0.0.0:8001/cube/microcerts
- Check the {{ modules|length }} is working correctly in the hero
- Check the margin on the bottom of the "share" button
- Check style linting

NOTE: An issue has been made to address the remaining comments, see [here](https://github.com/canonical-web-and-design/ubuntu.com/issues/9308)
